content/chat/rooms/index.textile

```diff
@@ -243,3 +279,19 @@ blang[react].
     return <div>Occupancy Component</div>;
   };
   ```
+
+blang[swift].
+  You can also create a subscribtion to the room status updates. An event will be emitted whenever the status of the room changes.
```

Comment:
If we agree to mitigate the difference between "registering a listener" and "creating a subscribtion" then sure it can be merged together.

Reply:
What do you recommend here @m-hulbert? Broadly, Swift and JS are doing the same thing - though in JS we tend to prefer the term listener, rather than subscription. Are the concepts close enough that we can just use the same term across the board?

I'd personally prefer we don't use different terms across the ecosystems, so its not confusing for users using both

Reply:
So what would be the final decision for this PR? @m-hulbert

Reply:
updated